oc_ftp_upload_worker/upload_worker.py

```
@@ -2,25 +2,81 @@
+                logging.debug('No new messages in queue.')
+                logging.debug('Sleeping [%s]' % self.sleep)
+                time.sleep(int(self.sleep))
+                continue
+            msg, msg_id = ds
+            logging.debug('new_msg_from_queue id [%s] is [%s]' % (msg_id, msg) )
+            client_code = msg[1][0]
+            logging.debug('client_code from message: [%s]' % client_code)
+            logging.debug('Calling upload_delivery')
+            try:
+                self.upload_delivery(client_code)
+            except Exception as e:
```

Ruff (0.13.2)

Do not catch blind exception: `Exception`

(BLE001)